lib/bundler/adder.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+require "net/http"
+require "yaml"
+
+module Bundler
+  class Adder
+    def initialize(name, version)
+      @name = name
+      @version = version || last_version_number
+    end
+
+    def add
+      Bundler.default_gemfile.open("a") do |f|
+        f.puts
+        f.puts output_line
+      end
+      Bundler.ui.confirm "Added to Gemfile: #{output_line}"
+    end
+
+  private
+
+    def last_version_number
+      uri = URI.parse("https://rubygems.org/api/v1/gems/#{@name}.yaml")
```

Comment:
Good for a baseline, but I believe there's existing Bundler code to handle these kinds of calls out to the API. (ex. how does `bundle outdated` do it?)